Code/src/main/resources/ui/style.css

```diff
@@ -7,95 +7,173 @@ CustomUIPane {
     -fx-font-size: 20px;
     -fx-font-weight: bold;
     -fx-font-family: Titillium;
-    -fx-background-color: aliceblue;
-    -fx-border-color: black;
+    -fx-text-fill: black;
     -fx-border-width: 2;
+    -fx-border-color: black;
 }
 
-.menu.block {
-    -fx-font-size: 12px;
+.error .block {
+    -fx-border-color: red;
 }
 
-.menu.block #categorySpace TitledPane {
-    -fx-padding: 3px;
+.block .header{
+    -fx-font-size: 20px;
+    -fx-font-weight: bold;
+    -fx-font-family: fantasy;
+    -fx-text-fill: black;
 }
 
-.menu.block #utilSpace {
-    -fx-padding: 3px;
-    -fx-spacing: 6px;
+.block .title {
+    -fx-font-family: Titillium;
```

Comment:
If you move the `-fx-font-family` declaration to `.block`, you might not have to mention it as often.